9.2/Dockerfile

```diff
@@ -24,14 +24,18 @@ EXPOSE 5432
 # to make sure of that.
 RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/postgresql92/epel-7-x86_64/download/rhscl-postgresql92-epel-7-x86_64.noarch.rpm && \
-    yum -y --setopt=tsflags=nodocs install gettext postgresql92 && \
+    yum -y --setopt=tsflags=nodocs --enablerepo=centosplus install gettext postgresql92 epel-release && \
+    yum -y --setopt=tsflags=nodocs install nss_wrapper && \
     yum clean all && \
+    chmod go+w /var/run/postgresql && \
```

Comment:
Just to follow up... the images built on the AWS vm work just fine with the chmod after everything, even when I pull the built images and run them locally on my boot2docker 1.6 vm. The issue happens only when I build the image locally. Moved the chmod to the single statement after things are copied in.